docs/contributing/accessibility/accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 5
+---
+
+# Accessibility
+
+Bitwarden strives to comply with [WCAG AA requirements](https://www.w3.org/WAI/WCAG2AA-Conformance).
+The following serves as an introduction to common aspects of accessibility that should be considered
+while implementing features and content. This list is not comprehensive, but provides a starting
+point for common “gotchas” that can limit the accessibility of a new feature. For comprehensive
+guidelines on accessibility, visit the official
+[WCAG site](https://www.w3.org/WAI/WCAG21/Understanding/).
+
+<bitwarden>
+
+:::info
+
+Additional resources on accessibility at Bitwarden can be found on
+[Confluence](https://bitwarden.atlassian.net/wiki/spaces/EN/pages/193953973).
+
+:::
+
+</bitwarden>
+
+## Visual Presentation
+
+| WCAG Success Criteria                                                                                                                                                                               | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [1.4.3 Contrast minimum](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)<br/><br/>[1.4.11 Non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) | Content should have contrast meeting or exceeding the WCAG AA guidelines:<br/>&nbsp;&nbsp;- For normal text: 4.5:1<br/>&nbsp;&nbsp;- For large text and graphics: 3:1<br/>&nbsp;&nbsp;- Use the [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/) to learn more.<br/>&nbsp;&nbsp;- When using variables from the Component Library, the `text-main` variable is tested on background and `background-alt` variables. All other backgrounds generally use `text-contrast`. |
```

Comment:
> When using variables from the Component Library, the `text-main` variable is tested on background and `background-alt` variables. All other backgrounds generally use `text-contrast`.

Should background be `background`? is it a variable or a class?

Would it be clearer to say something like: "In the Component Library, the `text-main` class has been tested on elements using the `background` and `background-alt` classes to ensure the text meets the proper contrast ratio. For text on all other backgrounds, you should generally use `text-contrast`."

Reply:
Yep, that's the case, I just missed a code block. 
I also like that wording better too!

Reply:
Actually, I'm going back on this one @JaredSnider-Bitwarden ; That makes more sense as a statement or standalone sentence, but in the context of a guideline I like the way the original sentence is worded that you only need to worry about this when using the component library.

Reply:
Ok, no worries :)